runtime/nls/cfre/cfrerr.nls

```diff
@@ -1520,3 +1520,10 @@ J9NLS_CFR_RECORD_CLASS_MUST_BE_FINAL.explanation=Please consult the Java Virtual
 J9NLS_CFR_RECORD_CLASS_MUST_BE_FINAL.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_RECORD_CLASS_MUST_BE_FINAL.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_LVTT_DOES_NOT_MATCH_LVT=LVTT entry does not match any LVT entry
```

Comment:
Please spell out LVTT and LVT in the error message.  It will be much clearer to end users what the issue is if we don't introduce acronyms

Reply:
I've updated the error message.